source/agent/quic/log4cxx.properties

```diff
@@ -66,19 +69,19 @@ log4j.logger.mcu.media.SoftVideoCompositor.AvatarManager=INFO
 log4j.logger.mcu.media.SoftVideoCompositor.SoftInput=INFO
 log4j.logger.mcu.media.SoftVideoCompositor.SoftFrameGenerator=INFO
 
-log4j.logger.woogeen.VCMFrameDecoder=INFO
-log4j.logger.woogeen.VCMFrameEncoder=INFO
-log4j.logger.woogeen.SVTHEVCEncoder=INFO
-log4j.logger.woogeen.FrameProcesser=INFO
+log4j.logger.owt.VCMFrameDecoder=INFO
```

Comment:
I think you can remove configuration items like 'VCMFrameDecoder' here, if you don't use them in quic-agent.

Reply:
Removed unnecessary iteams.